### PR TITLE
Improve SELECT and ENTRY text, plus ENTRY icon

### LINKED ISSUE #410: Icons for elements should reflect what they look like in the Sana App
- Entry should be a text field
- Selection should be a dropdown
- Multiple choice should be a multiple choice list
- Single Choice should be a radial button list

No idea for the last two...

### LINKED ISSUE #411: Element names should reflect what they actually are in the sana app
Entry -> Text
Selection -> Dropdown
Plugin (file) -> ???
Plugin (entry) -> ???

### PR DESCRIPTION
**Pull request by Kapin**
Closes #411
Closes #410

### CODE REVIEW

File: src-backbone/app/locales/en/translation.json [hunk 1]
```diff
@@ -103,13 +103,13 @@
...
-    "SELECT": "Selection",
+    "ENTRY": "Text Entry",
+    "SELECT": "Dropdown",
     "MULTI_SELECT": "Multiple Choice",
     "RADIO": "Single Choice",
     "PICTURE": "Picture",
-    "PLUGIN": "Plugin (file)",
-    "ENTRY_PLUGIN": "Plugin (entry)",
+    "PLUGIN": "Plugin",
+    "ENTRY_PLUGIN": "Entry Plugin",
```
Comment (Trinovantes) [inline src-backbone/app/locales/en/translation.json]: shouldn't this be "text entry plugin" then?

**Comment by Trinovantes** [comment]
are the others fine with the icon?

**Comment by Ommy** [comment]
"Yes" - @connorcimowsky , yes me

**Comment by Trinovantes** [comment]
then +1

File: src-backbone/app/locales/en/translation.json (hunk 1 above)
Comment (Kapin) [inline src-backbone/app/locales/en/translation.json]: I literally have no idea what this actually is in the app. I'd rather leave it as is for now.

**Pull request merged**